beacon-chain/blockchain/execution_engine.go

```diff
@@ -183,7 +183,7 @@ func (s *Service) getPayloadHash(ctx context.Context, root []byte) ([32]byte, er
 // notifyForkchoiceUpdate signals execution engine on a new payload.
 // It returns true if the EL has returned VALID for the block
 func (s *Service) notifyNewPayload(ctx context.Context, postStateVersion int,
-	postStateHeader *enginev1.ExecutionPayloadHeader, blk interfaces.SignedBeaconBlock) (bool, error) {
+	postStateHeader interfaces.ExecutionData, blk interfaces.SignedBeaconBlock) (bool, error) {
```

Comment:
What's the reason for changing `*enginev1.ExecutionPayloadHeader` to `interfaces.ExecutionData`. First glance, it seems unrelated to the PR title. Can you please explain it or even better cover that in the description?

Reply:
I updated the description